### PR TITLE
Ecbis release

### PR DESCRIPTION
**Pull request by maimoonak**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #[issue number] 

**Type**
Choose one: (Bug fix | Feature | Documentation | Testing | Code health | Release | Other) 

**Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [x] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginActivity.kt [hunk 1]
```diff
@@ -46,7 +46,7 @@ class LoginActivity :
     loginViewModel.apply {
       navigateToHome.observe(this@LoginActivity, { loginService.navigateToHome() })
       launchDialPad.observe(this@LoginActivity, { if (!it.isNullOrEmpty()) launchDialPad(it) })
-      loginUser()
+      // loginUser() commented out to make user login everytime
```
Comment (pld) [inline android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginActivity.kt:49]: why is this the desired functionality?

Comment (maimoonak) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginActivity.kt:49]: @dubdabasoduba requested this change to ensure that in field user always login when opening app

Comment (dubdabasoduba) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginActivity.kt:49]: @pld leaving it to always auto authenticate might lead to infringement of data privacy while on live usage.

Comment (pld) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginActivity.kt:49]: Gotcha, makes sense, should we add a todo here to make this configurable on a user or app bases, would that be useful?

Comment (maimoonak) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginActivity.kt]: added todo

Comment (pld) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginActivity.kt]: cool thanks

**Review by pld — APPROVED** [review]

**Review by pld — APPROVED** [review]

**Review by pld — APPROVED** [review]

**Pull request merged**